src/Arcus.Templates.WebApi/Startup.cs

```diff
@@ -50,7 +50,7 @@ namespace Arcus.Templates.WebApi
     public class Startup
     {
 #if Serilog
-        #warning Make sure that the appsettings.json is updated with your Azure Application Insights instrumentation key.
+#warning Make sure that the appsettings.json is updated with your Azure Application Insights instrumentation key.
```

Comment:
Woops, sorry for re-aligning all those compiler directives; must have something to do with my formatting options in vs.net

Reply:
Fixed this. :)